data/blog/2024-12-16-fall-2024-engr1a-cyber-lab.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Mastering Offensive Security, Building Our Own Modular Malware
+authors: [Samyukhtha Rajkumar Sridevi]
+category: Projects
+tags: [fall-2024, engr1a-cyber-lab]
```

Comment:
This is on me for not specifying the tags, but could we have a separate "cyber-lab" and a "engr-1a" tag? Thank you!